wazimap_ng/points/tasks.py

```diff
@@ -1,22 +1,13 @@
-import os
+import codecs
 import logging
 
 from django.db import transaction
-from django.db.models import Sum, FloatField
-from django.db.models.functions import Cast
-from django.contrib.postgres.fields.jsonb import KeyTextTransform
-from django.db.models import Count
-from django.db.models.query import QuerySet
 from django.conf import settings
 
-from . import models
 from .dataloader import loaddata
-from itertools import groupby
-from operator import itemgetter
 import pandas as pd
-from django_q.models import Task
-from wazimap_ng.general.services.permissions import assign_perms_to_group
 from wazimap_ng.general.services.csv_helpers import csv_logger
+from ..datasets.tasks.process_uploaded_file import detect_encoding
```

Comment:
let's move the encoding into https://github.com/OpenUpSA/wazimap-ng/blob/staging/wazimap_ng/utils.py

Reply:
Moved: https://github.com/OpenUpSA/wazimap-ng/pull/169/commits/2b66c86c09b6410d9177538871c8f6465afd3519